src/profile.c

```diff
@@ -126,6 +130,19 @@ void Profile_Print(RedisModule_Reply *reply, void *ctx) {
           RedisModule_ReplyKV_Double(reply, "Pipeline creation time",
             (double)(req->pipelineBuildTime / CLOCKS_PER_MILLISEC));
 
+      //Print total GIL time
+        if (profile_verbose){
+          if (RunInThread()){
+            RedisModule_ReplyKV_Double(reply, "Total GIL time", 
+            rs_timer_ms(&req->qiter.initTime));
```

Comment:
This is just the initial time, not the GIL time.
From a small test I did locally, I got:
```
7) Total GIL time
8) "1148342933.676"
```

IIUC, you already have the right time in `req->qiter.GILTime`